bin/ChakraCore/ChakraCore.vcxproj

```diff
@@ -166,6 +166,11 @@
       <Project>{706083f7-6aa4-4558-a153-6352ef9110f6}</Project>
     </ProjectReference>
   </ItemGroup>
+  <ItemGroup Condition="'$(BuildWabt)'=='true'">
+    <ProjectReference Include="..\..\lib\wabt\wabt.vcxproj">
```

Comment:
You can also put the condition on the ProjectReference to avoid the need for an extra ItemGroup